README.rst

```diff
@@ -29,19 +29,27 @@ How to install particular version: ::
 Features
 --------
 
-  * Concurrent requests (``--n_concurrent``)
-  * Pause/resume
-  * Gzip support
-  * Custom delimiters
-  * parallel processing
+* Concurrent requests (``--n_concurrent``)
+* Pause/resume
+* Gzip support
+* Custom delimiters
+* parallel processing
 
 
 Running batch_scoring
 ---------------------
 
-You can execute the batch_scoring command from the command line or you can pass parameters to a batch_scoring script from .ini file. Place the file in your home directory or the directory from which you are running the batch_scoring command. Use the syntax and arguments below to define the parameters. Note that if you run the script and also execute via the command line, the command line parameters take priority.
+You can execute the ``batch_scoring`` or ``batch_scoring_sse`` command from the command line or
+you can pass parameters to a ``batch_scoring`` or ``batch_scoring_sse`` script from .ini file. Place
+the file in your home directory or the directory from which you are running the ``batch_scoring``
+or ``batch_scoring_sse`` command. Use the syntax and arguments below to define the parameters.
+Note that if you run the script and also execute via the command line, the command line parameters take priority.
 
 The following table describes the syntax conventions; the syntax for running the script follows the table.
+Please note that we supply to different scripts:
```

Comment:
I'm not sure what you are trying to say here. Maybe:

Note that each command has a different application. Use: 

- ``batch_scoring`` to score using ORM on dedicated prediction instances.
- ``batch_scoring_sse`` to score on standalone prediction instances. If you are unsure of your instance type, contact DataRobot Support.

Two comments: 
1. Can you link "DataRobot Support" to "https://support.datarobot.com/hc/en-us"
2. ORM is an internal term that WE use to manage user usage. Can you just say "to score on dedicated prediction instances" or is ORM a separate case? If it is something else, ask Jeston how it applies to end users.